app/views/assets/_form.html.slim

```diff
@@ -22,7 +22,8 @@
       = f.label :assignee
       = f.text_field :assignee
     .field
-      = f.label :category
-      = f.text_field :category
+      = f.label :category_id
```

Comment:
Is this label going to say "Category ID"

If it is, maybe just write out the exact label that you want in normal HTML